Models/DTO/Administrator/FilterPharmaciesReportDTO.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.DTO.Administrator
+{
+    public class FilterPharmaciesReportDTO
+    {
+        [Required]
+        [DataType(DataType.DateTime)]
+        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
+        [JsonProperty("datefrom")]
```

Comment:
Maybe it would be more readable to serialize as `date-from`?

Reply:
Changed

Reply:
We can use just letters no special symbols.

Reply:
We can use only letters without special symbols.

Reply:
Maybe we could then use camel case at least to improve readability? (`dateFrom`)

Reply:
If we do it then when we wil need any other status we will have every time create one more Controller with one method.